src/renderer/wg_engine/tvgWgRenderData.h

```diff
@@ -70,15 +74,18 @@ class WgRenderDataShape: public WgRenderData {
 public:
     Array<WgGeometryData*> mGeometryDataShape;
     Array<WgGeometryData*> mGeometryDataStroke;
+    Array<WgGeometryData*> mGeometryDataImage;
```

Comment:
In most cases, shape render data doesn't include images. Ultimately, it requires more precise packed data and handling.
```
WgRenderDataImage : public WgRenderDataShape
```

Reply:
Good point. I have that idea too. But this case must be solved in more complex way: we need to made some refuctoring for pipelines, context and render targets. I believe that we will made it but not in this PR, bacouse it will impact to all rendering enteties, but not olny for images